backtrace-library/src/main/cpp/backends/crashpad-backend.cpp

```diff
@@ -17,6 +19,17 @@ bool InitializeCrashpad(jstring url,
                         jobjectArray attachmentPaths,
                         jboolean enableClientSideUnwinding,
                         jint unwindingMode) {
+    // Re-enable uploads if disabled
+    if (initialized && disabled) {
```

Comment:
This is very jarring to me. How can initialized be true if we're disabled? Please change the variable names or add a comment on how this is possible, because this is confusing for me.